src/components/post-form/PostForm.js

```diff
@@ -51,12 +53,18 @@ class PostForm extends Component {
     // });
   };
 
-  // todo 2: добавить функцию onReset которая будет очищать форму
+  // donetodo 2: добавить функцию onReset которая будет очищать форму
   //  подумайте как лучше сделать зачистку данных, если они хранятся  в стейте
+  onReset = () => {
+    this.setState({
+      title: '',
+      body: ''
+    });
+  };
 
   onSubmit = (e) => {
     e.preventDefault();
-
+    if (this.state.body === '' || this.state.title === '' ) return;
```

Comment:
also, `!this.state.body || !this.state.title` would be a valid condition
and also you can use `.trim()` to make it event better
but your variant is totally correct, just keep in mind for the next time